transient/configuration.py

```diff
@@ -190,6 +190,7 @@ class _TransientRunConfigSchema(_TransientConfigSchema):
     ssh_console = fields.Bool(missing=False)
     ssh_with_serial = fields.Bool(missing=False)
     shared_folder = fields.List(fields.Str(), missing=[])
+    drive_interface = fields.Str(missing="ide")
```

Comment:
As we are already defaulting to virtio networking for ssh, I think it's safe to default to virtio for this as well